Dockerfile

```diff
@@ -4,14 +4,20 @@ USER root
 WORKDIR /cvdi-stream
 
 # Add build tools.
-RUN apt-get update && apt-get install -y g++
+RUN apt-get update && apt-get install -y software-properties-common wget git make gcc-7 g++-7 gcc-7-base && update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 100 && update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-7 100
 
 # Install cmake.
 RUN apt install -y libprotobuf-dev protobuf-compiler
 RUN apt install -y cmake
 
 # Install librdkafka.
-RUN apt-get install -y libsasl2-dev libsasl2-modules libssl-dev librdkafka-dev
+RUN apt-get install -y sudo
```

Comment:
Do we need sudo here? I don't see it being used below?

Reply:
This is used on line 15 to run `apt-key add -`